docs/content/preview/faq/compatibility.md

```diff
@@ -147,7 +147,7 @@ YCQL is compatible with v3.4 of Apache Cassandra QL (CQL). Following questions h
 
 ### CQL features that are either unnecessary or disallowed in YCQL
 
-1. Lightweight transactions for compare-and-swap operations (such as incrementing integers) are unnecessary because YCQL achieves single row linearizability by default.
+1. Lightweight transactions for compare-and-set operations (such as incrementing integers) are unnecessary because YCQL achieves single row linearizability by default.
```

Comment:
1. Lightweight transactions for compare-and-set operations, such as incrementing integers, are unnecessary because YCQL achieves single-row linearizability by default.